libMBIN/Source/Template/NMSTemplate.cs

```diff
@@ -189,8 +189,8 @@ public class NMSTemplate
                     reader.BaseStream.Position = endPos;
                     return template;
                 default:
-                    if ( fieldType == "VariableStringSize") {
-                        reader.Align( 0x8 );
+                    if ( fieldType == "VariableStringSize") {  // typo: VariableSizeString
```

Comment:
Confused:  Which is good?

Reply:
I just added the comment, the change is left as an exercise for those who know if that if condition is even needed.